components/Feed/FeedList/FeedList.tsx

```
@@ -78,104 +83,130 @@ const FeedListItem = memo(function FeedListItem({
   );
 });
 
-export function FeedList({
-  items,
-  isLoading,
-  selectedItem,
-  onItemSelect,
-  savedScrollPosition = 0,
-}: FeedListProps) {
-  const scrollbarsRef = useRef<HTMLDivElement>(null);
+export function FeedList({}: FeedListProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)